src/fragments/lib/graphqlapi/existing-resources.mdx

```diff
@@ -19,7 +19,7 @@ Existing AWS AppSync resources can be used with the Amplify Libraries by referen
 ```
 
 - **API NAME**: Friendly name for the API (e.g., *api*)
-  - **endpoint**: The HTTPS endpoint of the AWS AppSync API (e.g. *https://aaaaaaaaaaaaaaaaaaaaaaaaaa.appsync-api.us-east-1.amazonaws.com/graphql*)
+  - **endpoint**: The HTTPS endpoint of the AWS AppSync API (e.g. *https://aaaaaaaaaaaaaaaaaaaaaaaaaa.appsync-api.us-east-1.amazonaws.com/graphql*). Custom domains can also be supplied here (e.g. *https://xxxxx.xxxxx.com/graphql*)
```

Comment:
nit: could replace xxx with a bit more meaningful example. e.g.

```
https://api.yourdomain.com/graphql
```

Reply:
Sure, that works!